frontend/components/portfolio/group/membership/MenuButton.vue

```diff
@@ -0,0 +1,175 @@
+<script setup lang="ts">
+import type OverlayPanel from 'primevue/overlaypanel'
+import { useToast } from 'primevue/usetoast'
+import { type Portfolio, type PortfolioGroup } from '@/openapi/generated/pacta'
+
+const { t } = useI18n()
+const toast = useToast()
+const { newPortfolioGroup: { newPortfolioGroupVisible } } = useModal()
+const pactaClient = usePACTA()
+
+interface Props {
+  portfolioGroups: PortfolioGroup[]
+  selectedPortfolios: Portfolio[]
+}
+const props = defineProps<Props>()
+interface Emits {
+  (e: 'changed-memberships'): void
+  (e: 'changed-groups'): void
+}
+const emit = defineEmits<Emits>()
+
+const prefix = 'components/portfolio/group/membership/MenuButton'
+const tt = (s: string) => t(`${prefix}.${s}`)
+const visible = useState<boolean>(`${prefix}.visible`, () => false)
+
+const overlayPanel = useState<OverlayPanel>(`${prefix}.overlayPanel`)
+const toggleMenu = (event: Event) => {
+  presentOrFileBug(overlayPanel.value).toggle(event)
+  visible.value = !visible.value
+}
+const hideMenu = () => {
+  visible.value = false
+}
+const showMenu = () => {
+  visible.value = true
+}
```

Comment:
nit: Normally I'm advocating for the opposite direction, but these are simple and straightforward enough that they could be inlined, since they're only used in one place.

Reply:
SG